src/Tribe/Ticket_Object.php

```diff
@@ -547,22 +556,32 @@ public function managing_stock( $manages_stock = null ) {
 		 * @return int
 		 */
 		public function inventory() {
+
+			/* @var Tribe__Cache $cache */
```

Comment:
:warning: This comment is 50% valid code; is this commented out code?

<sub>hash: 9022193b6c14712fcc95b54a4cbb6a7e</sub>